src/App.js

```diff
@@ -21,14 +21,16 @@ const EXAMPLE_TRACKS = [
   {"files": [{"type": "read", "name": "fakeReads"}]}
 ];
 
-function addTracksToColorSchemes(tracks, schemes) {
+function getColorSchemesFromTrack(tracks) {
```

Comment:
This gets all the schemes form all the tracks, so it should probably be `getColorSchemesFromTracks` plural.